components/event-bus/internal/knative/subscription/controller/subscription/reconcile_test.go

```diff
@@ -130,7 +131,7 @@ var testCases = []controllertesting.TestCase{
 				if _, ok := knSubscriptions[makeKnSubscriptionName(makeEventsActivatedSubscription())]; !ok {
 					t.Errorf("Knative subscription was NOT created")
 				}
-				if channel, ok := knChannels[makeKnChannelName(makeEventsActivatedSubscription())]; ok {
+				if channel, ok := knChannels[knChannelNames[makeKnChannelNamePrefix(makeEventsActivatedSubscription())]]; ok {
```

Comment:
An intermediary variable could make those calls slightly more readable I believe.

Reply:
on it

Reply:
done